src/pharmpy/modeling/common.py

```diff
@@ -128,6 +127,7 @@ def read_model_from_string(code, path=None):
         path = normalize_user_given_path(path)
         import pharmpy.workflows
 
+        # FIXME model.database should be removed
         model.database = pharmpy.workflows.default_model_database(path)
```

Comment:
What is the reason for keeping this in?

Reply:
Could it be removed already now?

Reply:
Should be okay! But then I guess the `path` argument doesn't make sense atm?

Reply:
I think path can be removed. Since it is not needed for the database any longer it wouldn't be needed to parse a model from a string.

Reply:
Fixed!

Reply:
Nice!